packages/@adobe/spectrum-css-temp/components/tray/index.css

```diff
@@ -66,6 +66,10 @@
   transition: opacity var(--spectrum-dialog-exit-animation-duration) cubic-bezier(0.5, 0, 1, 1) var(--spectrum-dialog-exit-animation-delay),
     transform var(--spectrum-dialog-exit-animation-duration) cubic-bezier(0.5, 0, 1, 1) var(--spectrum-dialog-exit-animation-delay);
 
+  &:last-child {
+    padding-bottom: calc(env(safe-area-inset-bottom) + 8px);
```

Comment:
while a really neat function that i just learned about (thank you!)
it seems it relies on a specific meta tag? https://developer.mozilla.org/en-US/docs/Web/CSS/env
> To tell the browser to use the whole available space on the screen, and so enabling us to use the env() variables, we need to add a new viewport meta value:
is this not the case?

Reply:
hmm well we don't have `viewport-fit=cover` in the docs site, just `width=device-width, initial-scale=1` and we still have this issue...

Reply:
You don't need the meta tag part. I've checked and I definitely see a difference in padding with and without the env() part.